### PR TITLE
add completion for sublime text

### PR DESCRIPTION
**Pull request by kloetzl**
Added completion for Sublime Text. Hope it is to you liking.

### CODE REVIEW

File: src/_subl [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+
+_subl() {
+  integer ret=1
+  local I="-h --help -v --version"
+  local -a args
+  args+=(
+    "(- *)"{-h,--help}'[Show help and exit]'
+    "(- *)"{-v,--version}'[Show version and exit]'
+    "($I)--project[Load the given project]:project:"
```
Comment (nicoulaj) [inline src/_subl]: There is a more elegant way for making `-h --help -v --version` mutually exclusive with all other options; see [_geany](https://github.com/zsh-users/zsh-completions/blob/master/src/_geany) for example

Comment (kloetzl) [inline src/_subl]: Thanks for the tip, but am I missing something? `--help<TAB>` is still getting completed after another command is already on the command line, which is not what I want.

zsh-completion-huh

Comment (nicoulaj) [inline src/_subl]: `geany --line 12 --help` is valid, you can try it, it is equivalent to running `geany --help`.

Comment (kloetzl) [inline src/_subl]: Hm, but then that is not what I want. Even though having `--help` after another option is technically valid, there is no real point in providing a completion. :thinking:

**Pull request merged**